.github/workflows/ci.yml

```diff
@@ -122,14 +125,11 @@ jobs:
 
       # python examples
       - uses: actions/setup-python@v2
-        if: runner.os == 'Linux' || runner.os == 'macOS'
         with:
           python-version: "3.10"
       - name: "Python Dataflow example"
-        if: runner.os == 'Linux' || runner.os == 'macOS'
```

Comment:
Looks like this errors on Windows: https://github.com/dora-rs/dora/actions/runs/6531652388/job/17733278897?pr=359#step:14:32

Reply:
Yeah, wanted to try it out, but it seems that the pip api is not the same on windows. I am going to remove this test from windows.